### ISSUE TITLE
Преобразование числа в текст и т.д.

### ISSUE THREAD

**Issue by gegemon2012** [BladeMight/Mahou#239]
Я смотрю что программа быстро развивается и хотелось бы внести несколько предложений:
1. Преобразование числа в текст - очень нужная функция для юристов, бухгалтеров и возможно людей других профессий.
2. Нужно упростить интерфейс. Слишком много настроек и это отпугивает пользователей от программы. Нужно больше всплывающих и подробных подсказок к некоторым настройкам так как все равно остаются вопросы. Может придумать переключение в упрощенный и продвинутый вид и в упрощенном оставить самое необходимое?
3. Очень хорошая идея с проверкой орфографии которую тут озвучили но и хорошо было бы автоматически исправлять ошибки в словах. 
4. Я понимаю что переводчик это неплохой бонус но есть же QTranslate который и так может почти всё...
5. Не нашел в словаре слова Firefox, Mahou. Я понимаю что невозможно объять необъятное но хотелось бы чтобы программа могла "понимать" такое как "bynth" - "интер", "afqthajrc" - "файерфокс". Я так понимаю что у программы нет никаких алгоритмов сопоставления и она ориентируется только по известным ей словам в словарях?

**Comment by gegemon2012** [comment]
Думаю что предложения под пунктом 2 можно проигнорировать.

**Comment by BladeMight** [comment]
> Я так понимаю что у программы нет никаких алгоритмов сопоставления и она ориентируется только по известным ей словам в словарях?

Да, сейчас только по полным словам, которые есть в словаре автозамен.

> Преобразование числа в текст - очень нужная функция для юристов, бухгалтеров и возможно людей других профессий.

Должно ведь работать для разных языков, верно? Значит нужно будет *еще* сделать словари для каждого.

> Нужно упростить интерфейс

Слишком многие уже предлагали, так что я отвечу как всегда - есть задумка над новым интерфейсом в Mahou 3.0, пока что в стадии "придумать".

> Я понимаю что переводчик это неплохой бонус но есть же QTranslate который и так может почти всё...

Не стану отрицать. Но там нет перевода на 2/3/4/5+ языков одновременно. В моём случае именно этого не хватало.

> хорошо было бы автоматически исправлять ошибки в словах

Это как? **При вводе** "не правильно"? Т.е. когда вводишь что-то неправильно. Или как?(**не при вводе**, но как-то по другому?)
Если при вводе, то на данный момент можно просто добавить пару(на самом деле очень много...) правил "неправильных" для **сниппетов**(не автозамен, работает иначе.), например:
```
->не правильно|yt ghfdbkmyj
====>неправильно<====
->почему то|gjxtve nj
====>почему-то<====
```

**Comment by Michael949498** [comment]
Встроенная возможность преобразования цифр в число прописью была бы очень кстати. Язык преобразования брать из текущей раскладки. Но вот просто словарем и заменой тут не обойтись. Для примера прикрепил скриншот программы, которой я пользуюсь на работе. Если получится такое повторить -- отлично. :)
![image](https://user-images.githubusercontent.com/45428320/60613072-ced0e380-9dd2-11e9-9002-62c493dd93af.png)
